core/trino-main/src/test/java/io/trino/block/BenchmarkRowBlockBuilder.java

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Remove benchmark results from this commit (and move to other one). Could you format results in such a way that it's easy to compare results before/after change?